src/requestforattestation/RequestForAttestation.ts

```diff
@@ -99,6 +116,29 @@ export default class RequestForAttestation implements IRequestForAttestation {
     this.claimerSignature = this.sign(identity)
   }
 
+  /**
+   * Removes [[Claim]] properties from the [[RequestForAttestation]] object.
```

Comment:
Could we explain briefly when/why this is useful (privacy/...) - and maybe how it relates to `createPresentation`?

Reply:
Added the extra detail, makes it 100% better.